python/py-ubjson/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-ubjson
+version             0.16.1
+platforms           darwin
+license             Apache-2
+maintainers         {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
+
+description         Universal Binary JSON encoder/decoder
+long_description    ${description}
+
+homepage            https://github.com/Iotic-Labs/py-ubjson
+
+checksums           rmd160  c5d75a21a18b0786924414987d8a568f593b20cf \
+                    sha256  b9bfb8695a1c7e3632e800fb83c943bf67ed45ddd87cd0344851610c69a5a482 \
+                    size    50316
+
+distname            py-ubjson-${version}
+master_sites        pypi:p/py-ubjson
+
+python.versions     27 38
```

Comment:
Is py27 needed for something? I think we should not be adding new ports for old python versions.

Reply:
You're right, my fault. Removed.